package.json

```diff
@@ -6,7 +6,7 @@
     "adal-angular": "1.0.15",
     "ag-grid": "16.0.1",
     "ag-grid-react": "16.0.0",
-    "camelcase-keys": "4.2.0",
+    "camelcase-object": "1.0.0",
```

Comment:
So was build failure fixed just by moving to `camelcase-object`?

Reply:
Yep. `camelcase-keys` was written to be used with node, not in the browser. So babel was freaking out because of es5 issues. `camelcase-object` works correctly in the browser.